client/cmd/certs.go

```diff
@@ -93,48 +118,117 @@ func CertAdd(cert, key, commonName, sans string) error {
 	return nil
 }
 
-func processCertsAdd(c *client.Client, cert, key, commonName, sans string) error {
-	if sans != "" {
-		for _, san := range strings.Split(sans, ",") {
-			if err := doCertAdd(c, cert, key, san); err != nil {
-				return err
-			}
-		}
-		return nil
+func doCertAdd(c *client.Client, cert string, key string, name string) error {
+	certFile, err := ioutil.ReadFile(cert)
+	if err != nil {
+		return err
+	}
+
+	keyFile, err := ioutil.ReadFile(key)
+	if err != nil {
+		return err
 	}
 
-	return doCertAdd(c, cert, key, commonName)
+	_, err = certs.New(c, string(certFile), string(keyFile), name)
+	return err
 }
 
-func doCertAdd(c *client.Client, cert string, key string, commonName string) error {
-	certFile, err := ioutil.ReadFile(cert)
+// CertRemove deletes a cert from the controller.
+func CertRemove(name string) error {
+	c, err := client.New()
+	if err != nil {
+		return err
+	}
 
+	fmt.Printf("Removing %s... ", name)
+	quit := progress()
```

Comment:
Don't know - Not my original code, even tho it looks like it for some reason in the diff. https://github.com/deis/workflow/blame/master/client/cmd/certs.go#L135